mix_helpers.exs

```diff
@@ -136,6 +136,7 @@ defmodule Mix.Appsignal.Helper do
     defp map_arch('x86_64-pc-linux-gnu' ++ _), do: "x86_64-linux"
     defp map_arch('x86_64-redhat-linux-gnu' ++ _), do: "x86_64-linux"
     defp map_arch('x86_64-unknown-linux' ++ _), do: "x86_64-linux"
+    defp map_arch('x86_64-pc-linux-musl' ++_), do: "x86_64-linux"
```

Comment:
we can probably drop the `-musl` suffix here and match more host triples. We can maybe also add an `i686-pc-linux` host triple to this.

WDYT @jeffkreeftmeijer ?